dag-cbor-derive/Cargo.toml

```diff
@@ -8,14 +8,13 @@ edition = "2018"
 proc-macro = true
 
 [dependencies]
-async-trait = "0.1"
 proc-macro2 = "1.0"
 quote = "1.0"
 syn = "1.0"
 synstructure = "0.12.0"
 
 [dev-dependencies]
-async-std = "0.99"
+libipld = { path = "../" }
```

Comment:
From what I can tell `dag_cbor_derive` uses `libipld` which it can't find without this.

Reply:
the examples are mainly used here:
```rust
#[cfg(test)]
mod tests {
    #[test]
    fn test() {
        let t = trybuild::TestCases::new();
        t.pass("examples/basic.rs");
        t.pass("examples/name_attr.rs");
        //t.pass("examples/repr_attr.rs");
    }
}
```

Reply:
I do get it but type-checker seemed to complain about `libipld` not being in the scope. From what I gathered it was due to the fact that `libipld` was not resolved in the code below:

```rust
fn dag_cbor_derive(s: Structure) -> TokenStream {
    let write_cbor = gen::write_cbor(&s);
    let read_cbor = gen::read_cbor(&s);
    s.gen_impl(quote! {
        use libipld::cbor::IpldError;
        use libipld::cbor::{ReadCbor, WriteCbor, CborResult as Result};
        use libipld::cbor::encode::{write_u64, Write};
        use libipld::cbor::decode::{read_u8, read_key, Read};

        gen impl WriteCbor for @Self {
            #write_cbor
        }

        gen impl ReadCbor for @Self {
            #read_cbor
        }
    })
}
```

Reply:
No it's a mistake your completely right, I just figured that that's the reason I didn't notice it.